apps/music/js/utils.js

```diff
@@ -186,3 +190,22 @@ function createListElement(option, data, index, highlight) {
 
   return li;
 }
+
+function showImage(image, url, option) {
+  // We could probably have many animation options.
+  // Add them here in the future.
+  var isFadeIn = (option === 'fadeIn');
+  image.classList.toggle('fadeIn', !isFadeIn);
```

Comment:
I don't think this does what you want. Two reasons:

1. If you don't specify `'fadeIn'` for `option`, this line will set the `fadeIn` style on the image.
2. This change is happening *after* the image has been fetched from the database; previously, the change occurred *before* it had been fetched.